### PR TITLE
RA-1604: Confirmation before deleting a condition

### PR DESCRIPTION
**Pull request by haripriya999**
# RA-1604: Confirmation before deleting a condition

## Description of what I changed
Confirm if a condition has to be deleted before the action is performed

## Issue I worked on

see https://issues.openmrs.org/browse/RA-1604

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by dkayiwa** [comment]
On the ticket, can you add screenshots with the effects of your changes? Also include a link to the talk thread where this was discussed.

**Comment by haripriya999** [comment]
I added comments on the ticket and updated the description :)

File: omod/src/main/webapp/resources/scripts/conditionlist/controllers/manageconditions.controller.js [hunk 1]
```diff
@@ -41,33 +41,38 @@ function ManageConditionsController($scope, RestfulService, CommonFunctions) {
...
+
 
     self.saveCondition = self.saveCondition || function (condition) {
             var conditions = [];
             conditions.push(condition);
             RestfulService.post('condition', conditions, function (data) {
-                //emr.successAlert("conditionlist.updateCondition.success");
-                emr.successAlert("Condition Saved Successfully");
+                if(condition.voided == false){
+                    //emr.successAlert("conditionlist.updateCondition.success");
```
Comment (dkayiwa) [inline omod/src/main/webapp/resources/scripts/conditionlist/controllers/manageconditions.controller.js]: What is the above comment used for?

**Comment by dkayiwa** [comment]
A dangerous button should not be the default.

**Comment by dkayiwa** [comment]
Secondly, instead of that dialog box, can't you use the one like for removing an allergy?

**Comment by haripriya999** [comment]
> Secondly, instead of that dialog box, can't you use the one like for removing an allergy?

I will make the required changes

**Comment by haripriya999** [comment]
> Secondly, instead of that dialog box, can't you use the one like for removing an allergy?

I updated the PR and attached the latest screenshots in the talk link - https://talk.openmrs.org/t/tracking-progress-of-the-condition-list-phase-gsoc-project/23400/28

**Comment by dkayiwa** [comment]
@haripriya999 are you still working on this?

**Comment by haripriya999** [comment]
> @haripriya999 are you still working on this?

@dkayiwa  This PR has been completed.

**Comment by dkayiwa** [comment]
@haripriya999 do you have screenshots with the effects of your changes attached on the ticket?

**Comment by haripriya999** [comment]
@dkayiwa  I updated the screenshots

**Pull request merged**